src/content/docs/guides/policy.md

```diff
@@ -9,15 +9,15 @@ This guide will help you understand what a LavaMoat Policy is and how to use it.
 
 ## What's a Policy?
 
-A Policy is an object that describes which _resources_ any given dependency can access. These "resources" include globals, builtins (e.g., `node:fs`), native modules, and other packages. _All direct and transitive dependencies_ in your application's dependency tree are subject to a Policy.
+A Policy is an object that describes which _powers_ any given dependency can access. These "powers" include globals, builtins (e.g., `node:fs`), native modules, and other packages. _All direct and transitive dependencies_ in your application's dependency tree are subject to a Policy.
```

Comment:
```suggestion
A Policy is an object that describes which _capabilities_ any given dependency can access. These capabilities include globals, builtins (e.g., `node:fs`), native modules, and other packages. _All direct and transitive dependencies_ in your application's dependency tree are subject to a Policy.
```
Why "powers"? Is this an existing concept from elsewhere or minted here?

Reply:
It's a concept from Object Capability programming and I also defined iit in the glossary now.
You can't frequent Endo meetings to become more familiar with the terminology, but they're recorded and you can catch up on them on Agoric's youtube.
Here's an archive https://www.youtube.com/watch?v=tM5NyB7xxYM&list=PLzDw4TTug5O0eUj81Vnkp-mFuI4O0rBnc
Not sure if the latest ones were being added to the playlist. Check main channel.

Reply:
I'll shave the yak on the difference between capabilities and powers internally